pkg/kubeapiserver/options/authentication.go

```diff
@@ -37,13 +37,15 @@ import (
 	cliflag "k8s.io/component-base/cli/flag"
 	"k8s.io/klog/v2"
 	openapicommon "k8s.io/kube-openapi/pkg/common"
+
 	serviceaccountcontroller "k8s.io/kubernetes/pkg/controller/serviceaccount"
 	"k8s.io/kubernetes/pkg/features"
 	kubeauthenticator "k8s.io/kubernetes/pkg/kubeapiserver/authenticator"
 	authzmodes "k8s.io/kubernetes/pkg/kubeapiserver/authorizer/modes"
 	"k8s.io/kubernetes/plugin/pkg/auth/authenticator/token/bootstrap"
 )
 
+// BuiltInAuthenticationOptions contains all build-in authentication options for APIServer
```

Comment:
don't we either write "API server" or "apiserver" elsewhere?

Reply:
I search the code, "API Server" is most used